### PR TITLE
Android: Add Indonesian Translation

### PR DESCRIPTION
**Pull request by maybelucid**

### CODE REVIEW

File: build_android/src/main/res/values-id/strings.xml [hunk 1]
```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<!-- Application Navigation -->
+
+	<string name="about_play">Tentang Play!</string>
+	<string name="about_play_emu">Play! Emulator PlayStation 2</string>
```
Comment (Croden1999) [inline build_android/src/main/res/values-id/strings.xml:7]: here should be `Play! PlayStation 2 Emulator`

**Review by Croden1999 — CHANGES_REQUESTED** [review]

File: build_android/src/main/res/values-id/strings.xml (hunk 1 above)
Comment (maybelucid) [reply build_android/src/main/res/values-id/strings.xml:7]: got it

File: build_android/src/main/res/values-id/strings.xml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	<string-array name="pref_emu_video_gs_handler_entries">
+		<item>OpenGL</item>
+		<item>Vulkan (Eksperimental)</item>
+	</string-array>
+
+	<string-array name="pref_emu_video_res_factor_entries">
+		<item>1x</item>
+		<item>2x</item>
+		<item>4x</item>
+		<item>8x</item>
```
Comment (Croden1999) [inline build_android/src/main/res/values-id/strings.xml:123]:
```suggestion
```

also here

File: build_android/src/main/res/values-id/strings.xml [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+	</string-array>
+
+	</string-array>
+
+	<string-array name="pref_emu_video_presentation_mode_entries">
+		<item>Isi Layar</item>
+		<item>Paskan layar(Default)</item>
+		<item>Ukuran asli</item>
+	</string-array>
+
```
Comment (Croden1999) [inline build_android/src/main/res/values-id/strings.xml:126]:
```suggestion
```

**Review by Croden1999 — CHANGES_REQUESTED** [review]

**Comment by Croden1999** [comment]
please Mr. @jpd002 check this pr.

**Pull request merged**